src/masonite/inertia/core/InertiaResponse.py

```diff
@@ -44,10 +43,11 @@ def _load_routes(self):
             if route.named_route:
                 self.routes.update({route.named_route: route.route_url})
 
-    def render(self, component, props={}, custom_root_view=None):
+    def render(self, component, props={}, custom_root_view="app"):
```

Comment:
A trick with packages is always setting sensible defaults. Since the default should really be an app.js file. I added that as the root file

Reply:
I agree ! here this argument is used to define the .html template file and the default value is defined as "app" in the default inertia config. But it might be better to do it here and remove the default value in inertia config